# Name of the joints to be controlled.
control_joints: [
  'rh_FFJ3', 'rh_MFJ3', 'rh_RFJ3', 'rh_LFJ3', 'rh_THJ5'
]

# For each joint, its topic, initial (open) position, pregrasp postion,
# and minimum and maximum joint limits should be specified.

rh_FFJ0: {
  'topic': '/sh_rh_ffj0_position_controller/command',
  'init': 0.349, 'pregrasp': 0.6, 'min': 0.349, 'max': 1.4
}
rh_FFJ3: {
  'topic': '/sh_rh_ffj3_position_controller/command',
  'init': 0.0, 'pregrasp': 0.0, 'min': 0.0, 'max': 1.3
}

rh_MFJ0: {
  'topic': '/sh_rh_mfj0_position_controller/command',
  'init': 0.349, 'pregrasp': 0.6, 'min': 0.349, 'max': 1.4
}
rh_MFJ3: {
  'topic': '/sh_rh_mfj3_position_controller/command',
  'init': 0.0, 'pregrasp': 0.0, 'min': 0.0, 'max': 1.3
}

rh_RFJ0: {
  'topic': '/sh_rh_rfj0_position_controller/command',
  'init': 0.349, 'pregrasp': 0.6, 'min': 0.349, 'max': 1.4
}
rh_RFJ3: {
  'topic': '/sh_rh_rfj3_position_controller/command',
  'init': 0.0, 'pregrasp': 0.0, 'min': 0.0, 'max': 1.4
}

rh_LFJ0: {
  'topic': '/sh_rh_lfj0_position_controller/command',
  'init': 0.349, 'pregrasp': 0.6, 'min': 0.349, 'max': 1.4
}
rh_LFJ3: {
  'topic': '/sh_rh_lfj3_position_controller/command',
  'init': 0.0, 'pregrasp': 0.0, 'min': 0.0, 'max': 1.4
}

rh_THJ2: {
  'topic': '/sh_rh_thj2_position_controller/command',
  'init': 0.0, 'pregrasp': 0.35, 'min': 0.0, 'max': 0.5
}
rh_THJ3: {
  'topic': '/sh_rh_thj3_position_controller/command',
  'init': 0.02, 'pregrasp': 0.02, 'min': 0.02, 'max': 0.08
}
rh_THJ4: {
  'topic': '/sh_rh_thj4_position_controller/command',
  'init': 0.19, 'pregrasp': 0.87, 'min': 0.19, 'max': 1.10
}
rh_THJ5: {
  'topic': '/sh_rh_thj5_position_controller/command',
  'init': -0.15, 'pregrasp': -0.15, 'min': -0.15, 'max': 0.6
}
